src/application/services/useNote.ts

```diff
@@ -159,6 +166,19 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     parentNote.value = response.parentNote;
   }
 
+  /**
+   * Returns list of tools used in the note
+   *
+   * @param content - content of the note
+   */
+  async function resolveToolsByContent(content: NoteContent): Promise<NoteTool[]> {
+    const { allTools } = useTools(noteTools);
+
+    return content.blocks.map((block) => {
+      return { name: block.type, id: (allTools.value ?? []).find((tool) => tool.name === block.type)!.id };
```

Comment:
its hard to read, please, write it via the separate statements

Reply:
➕